lib/octobox/configurator.rb

```diff
@@ -44,7 +44,7 @@ def github_app
     attr_writer :github_app
 
     def fetch_subject
-      @fetch_subject || (ENV['FETCH_SUBJECT'].try(:downcase) == "true")
+      @fetch_subject || env_boolean('FETCH_SUBJECT')
```

Comment:
`env_boolean` helper that allows switching options on or off by passing `true/false` rather than toggling via value/blank